clonelists/Sony - PlayStation 2 (Redump).json

```diff
@@ -2846,6 +2852,24 @@
 				{"searchTerm": "Guitar Hero Encore - Rocks the 80s"}
 			]
 		},
+		{
+			"group": "GuitarFreaks & DrumMania V3",

```

Comment:
This won't work. The group name for both _GuitarFreaks & DrumMania V3_ and _GuitarFreaks V2 & DrumMania V2_ is `guitarfreaks & drummania`. You can see this by putting your proposed group name in the title tool. **File > Title tool**.

These are better off living in the overrides, using similar patterns to Micro Machines v4 and NBA Street V3.